### PR TITLE
Add `roman-numerals` exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/practice/roman-numerals/.meta/example.ua [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Hundreds      ← {"" "C" "CC" "CCC" "CD" "D" "DC" "DCC" "DCCC" "CM"}
+Tens          ← {"" "X" "XX" "XXX" "XL" "L" "LX" "LXX" "LXXX" "XC"}
+Ones          ← {"" "I" "II" "III" "IV" "V" "VI" "VII" "VIII" "IX"}
+Lookup        ← comptime(☇ 0 ≡≡(⍚/◇⊂) ⊞(⊂⊂) Hundreds Tens Ones)
```
Comment (ErikSchierboom) [inline exercises/practice/roman-numerals/.meta/example.ua:4]: I did not expect compile time support, but it is supported!

Comment (vaeng) [reply exercises/practice/roman-numerals/.meta/example.ua:4]: I saw that but had no inspiration what to do with it. Now I've got to look it up closely

Comment (ErikSchierboom) [reply exercises/practice/roman-numerals/.meta/example.ua:4]: Did you mean not to approce this PR?

Comment (vaeng) [reply exercises/practice/roman-numerals/.meta/example.ua:4]: I meant the `comptime` function

**Review by vaeng — APPROVED** [review]

File: exercises/practice/roman-numerals/.meta/example.ua (hunk 1 above)
Comment (Marcos-cat) [reply exercises/practice/roman-numerals/.meta/example.ua:4]: `Lookup ← ♭ ⊞⍚(⊂⊂) Hundreds Tens Ones`

**Pull request merged**